### PR TITLE
Issue #1260 Added Free/Busy information

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added Free/Busy information #1260

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

**Comment by j0rt3g4** [comment]
Solving Issue #1260

File: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md [hunk 1]
```diff
@@ -78,6 +78,16 @@ The agent supports outbound unauthenticated proxy servers but requires additiona
 
 The HCW establishes connections from your Client Access Server to domains.live.com to exchange metadata and establish trusts. Because connections originate from your CAS server, the proxy settings on that server (from `Get-ExchangeServer | Format-List InternetWebProxy`) must be set correctly or outbound free/busy can fail. In addition to connection failures, the HCW won't be able to configure delegated authentication if the proxy setting is incorrect.
 
+## Free/Busy Requirements
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md]:
```suggestion
## Free/Busy requirements
```

File: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md [hunk 2]
```diff
@@ -78,6 +78,16 @@ The agent supports outbound unauthenticated proxy servers but requires additiona
...
+
+The hybrid wizard automatically handles the Free/Busy information required to work in both environments (cloud and on-premises).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md]:
```suggestion
The hybrid wizard automatically handles the free/busy information required to work in both environments (cloud and on-premises). 
```

File: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md [hunk 3]
```diff
@@ -78,6 +78,16 @@ The agent supports outbound unauthenticated proxy servers but requires additiona
 
 The HCW establishes connections from your Client Access Server to domains.live.com to exchange metadata and establish trusts. Because connections originate from your CAS server, the proxy settings on that server (from `Get-ExchangeServer | Format-List InternetWebProxy`) must be set correctly or outbound free/busy can fail. In addition to connection failures, the HCW won't be able to configure delegated authentication if the proxy setting is incorrect.
 
+## Free/Busy Requirements
+
+The hybrid wizard automatically handles the Free/Busy information required to work in both environments (cloud and on-premises). 
+
+In case that it is required to configure the free/busy information with a 3rd party company, please go to EAC/Organization and configure the individual or organization sharing between the two systems (Exchange Online / Exchange On-premises or Exchange On-premises - Exchange On-premises).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md]:
```suggestion
If it is required to configure the free/busy information with a third party company, go to EAC/Organization and configure the individual or organization sharing between the two systems (Exchange online/Exchange on-premises or Exchange on-premises/Exchange on-premises).
```

File: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md [hunk 4]
```diff
@@ -78,6 +78,16 @@ The agent supports outbound unauthenticated proxy servers but requires additiona
...
+ 
+If you have problems or need to troubleshoot the free/busy information, please go to [this link](https://support.microsoft.com/help/2555008/how-to-troubleshoot-free-busy-issues-in-a-hybrid-deployment-of-on-prem).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md]:
```suggestion
Should you encounter any problems or need to troubleshoot the free/busy information, refer to [How to troubleshoot free/busy issues in a hybrid deployment of on-premises Exchange Server and Exchange Online in Office 365](https://support.microsoft.com/help/2555008/how-to-troubleshoot-free-busy-issues-in-a-hybrid-deployment-of-on-prem).
```

**Comment by j0rt3g4** [comment]
@scanum

**Pull request merged**